charts/coredns/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: coredns
-version: 1.30.0
+version: 1.30.1
```

Let's up the minor version, we're adding stuff not just fixing

```suggestion
version: 1.31.0
```